Assets/HoloToolkit/Input/Scripts/Voice/KeywordManager.cs

```diff
@@ -130,5 +132,31 @@ public void StopKeywordRecognizer()
                 keywordRecognizer.Stop();
             }
         }
+
+        private void OnPhraseRecognized(UnityEngine.Windows.Speech.ConfidenceLevel confidence,
+            TimeSpan phraseDuration, DateTime phraseStartTime,
```

Comment:
NIT: I suggest putting all input params on the same line, or all on separate line.

Also, why I'd suggest calling this method from KeywordRecognizer_OnPhraseRecognized, so that we only have one entry point to RaisePhraseRecognizedEvent